lib/SearchAndSort/SearchAndSortQuery.js

```diff
@@ -439,9 +439,15 @@ class SearchAndSortQuery extends React.Component {
     }
 
     const sortOrder = orders.slice(0, maxSortKeys).join(',');
+    const sortFields = { sort: sortOrder };
 
-    // this.log('action', `sorted by ${sortOrder}`);
-    this.callQuerySetter({ sort: sortOrder });
+    this.internalSetState(
+      { sortFields },
+      'sort.change',
+      [
+        () => this.callQuerySetter(sortFields)
+      ]
```

Comment:
Do we need the `sortChangeCallback` here? e.g. 
```
      [
        sortChangeCallback,
        () => this.callQuerySetter(sortFields)
      ]
```

Reply:
@zburke good catch! It will add it there.